composer.json

```diff
@@ -1,46 +1,46 @@
 {
-  "name": "laravel-doctrine/migrations",
-  "type": "library",
-  "description": "Doctrine Migrations for Laravel",
-  "license": "MIT",
-  "keywords": [
-    "doctrine",
-    "laravel",
-    "orm",
-    "data mapper",
-    "database",
-    "migrations"
-  ],
-  "authors": [
-    {
-      "name": "Patrick Brouwers",
-      "email": "patrick@maatwebsite.nl"
+    "name": "laravel-doctrine/migrations",
+    "type": "library",
+    "description": "Doctrine Migrations for Laravel",
+    "license": "MIT",
+    "keywords": [
+        "doctrine",
+        "laravel",
+        "orm",
+        "data mapper",
+        "database",
+        "migrations"
+    ],
+    "authors": [
+        {
+            "name": "Patrick Brouwers",
+            "email": "patrick@maatwebsite.nl"
+        }
+    ],
+    "require": {
+        "php": "^7.4|^8",
```

Comment:
laravel-doctrine/orm 2.0 only supports PHP 8+ :)

Reply:
That's great :grin:
[930e094](https://github.com/laravel-doctrine/migrations/pull/135/commits/930e094cc171d3b5a09c6c6398caa0f480a1791f)